### PR TITLE
Fix editing messages containing replies in the message builder

### PR DESCRIPTION
**Pull request by VelvetThePanda**
Fix modifying messages throwing an exception when using the DiscordMessageBuilder

# Notes
I'm 99% sure I fixed this before

### CODE REVIEW

**Comment by Neuheit** [comment]
If we're removing this we should probably add a disclaimer to the `WithReply()` method stating that they can't modify a reply.

**Comment by VelvetThePanda** [comment]
Sounds good

**Comment by VelvetThePanda** [comment]
@Neuheit should be good, I think.

File: DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs [hunk 1]
```diff
@@ -395,13 +396,6 @@ internal void Validate(bool isModify = false)
...
                 throw new ArgumentException("A message can only have up to 10 embeds.");
 
-            if (isModify)
-            {
-                if (this.ReplyId.HasValue)
-                    throw new ArgumentException("You cannot change the ReplyID when modifying a message");
-            }
-            else
-            {
                 if (this.Files?.Count == 0 && string.IsNullOrEmpty(this.Content) && (!this.Embeds?.Any() ?? true) && this.Sticker is null)
```
Comment (Emzi0767) [inline DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs:399]: If the surrounding scope bracket was remvoed, this should be unindented. Am I missing something?

**Review by Emzi0767 — CHANGES_REQUESTED** [review]

**Review by Emzi0767 — APPROVED** [review]

**Pull request merged**